Composer/scripts/l10nCheck.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable security/detect-non-literal-fs-filename */
+
+const fs = require('fs');
+const path = require('path');
+
+const localePath = 'packages/server/src/locales';
```

Comment:
does this need to be normalized for Windows?

Reply:
I'm running this on a Windows machine and it worked fine.